### DISCUSSION TITLE
ADD COLUMN Migration, column already exists

### DISCUSSION THREAD

**Discussion by TrofinSorin** [sequelize/sequelize#16687] [Help & Questions]
I have a few migrations, is there a way when running sequelize-cli db:migrate to check in addColumn if it already exists to not do anything and continue the other migrations?

I see something has been done here, but didn't see anything in the documentation, is it in version 7?

https://github.com/sequelize/sequelize/pull/15119

**Comment by ephys** [comment]
Yep! It's part of v7 https://sequelize.org/api/v7/interfaces/_sequelize_core.index.addcolumnoptions#ifNotExists
